views/development/all-templates.njk

```diff
@@ -6,10 +6,34 @@
 {% extends "shared/base.njk" %}
 {% set pageTitleKey = "All templates" %}
 
+{% if errorState %}
+    {% set errorText = 'Select a page and a context if appropriate.' %}
+{% endif %}
+
 {% block content %}
   <form id="pageSelectForm" action="/dev/all-templates" method="POST">
     <input type="hidden" name="_csrf" value="{{ csrfToken }}">
 
+    {% set templateRadioConfig = [] %}
+    {% for page, contexts in templatesWithContextRadioOptions %}
+      {% set conditional = {
+        html: govukRadios({idPrefix: "pageContext",
```

Comment:
Looks like the idPrefix isn't enough to make the context radios unique, which is breaking a couple of things, e.g. being able to click the label to select the button.

You might need to add the page id into this prefix as well